### PR TITLE
Portfolio accessiblity

### PR DESCRIPTION
**Pull request by essiewanjiru99**
In this PR, we checked for the following accessibilty topics; 

Page Title - Change the title of the page
Image Text Alternative - Provide a descriptive text for important images
Text headings - No issues found
Color contrast -No issues found
These are the links to the corrected lines;
https://github.com/essiewanjiru99/essie-portfolio/compare/essiewanjiru99:393a15d...essiewanjiru99:219e893#
https://github.com/essiewanjiru99/essie-portfolio/compare/essiewanjiru99:393a15d...essiewanjiru99:219e893#
https://github.com/essiewanjiru99/essie-portfolio/blob/238cb7233a50e3f89c7a7d0d76af2a95c171a1fd/index.html#L62

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Hi @essiewanjiru99 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a pair programming project with one contributor. Kindly ensure that your partner commits to the PR equally for it to be valid.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>My Portfolio Website</title>
```
Comment (bobb-Rob) [inline index.html:7]: - [ ] The page titles you provided have room for improvement. Here are revised versions that better describe the content of the page:
  - Poor titles:

      - Microverse project | Your portfolio.
      - Portifolio project.

  - Better page titles:

      - Mark's Portfolio | Full-Stack Web Developer.
      - Mark's Professional Portfolio | Full-Stack Web Development.

By including your name and specifying your role as a full-stack web developer, the revised titles provide clearer and more informative descriptions of the page content. Kindly implement this.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @essiewanjiru99, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Johnadibe) [reply index.html:7]: > - [ ] In response to the previous code reviewer's request, it is suggested that you make improvements to the page title you provided. The revised version mentioned above will offer clearer and more informative descriptions of the content on the page. Using this updated title will enhance accessibility and improve the understanding of the page's purpose.☺

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️
### Hi @essiewanjiru99 and @Akorede3133🙋‍♂️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- Descriptive PR✔
- Provided Image text alternatives✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Johnadibe — APPROVED** [review]
# Status: Approved ✔️
### Hi @essiewanjiru99 and @Akorede3133👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

LeonardoInGatsby

## Highlights
- Implemented the change requested by the previous reviewer✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**